src/bindings/python/src/openvino/frontend/pytorch/decoder.py

```diff
@@ -16,7 +16,10 @@ def get_type_from_py_type(value):
     if isinstance(value, float):
         return OVType.f32
     if isinstance(value, int):
-        return OVType.i32
+        # Python int is 64 bit, but we will convert it to int32 except cases when it can't fit in 32 bits
+        if torch.iinfo(torch.int).min <= value <= torch.iinfo(torch.int).max:
+            return OVType.i32
+        return OVType.i64
```

Comment:
why can't we convert it as? Can it be a problem that some operand will be decoded as `int32` and the other operand as `int64` and we will have type inconsistency?

Reply:
We had discussion with @slyalin about this issue, he was in favor of converting to int32 when we can do it, to reduce conversions later